src/app/modules/browse/browse.tpl.html

```diff
@@ -44,7 +44,12 @@
                 <div class="container-fluid">
                     <div class="searchRow row">
                         <div class="col-md-6" style="padding-bottom:10px">
-                            <div ng-show="initialized">{{bookMessage}}</div>
+                            <div ng-show="initialized">
```

Comment:
Should we hide this whole thing if there are no hidden books? (e.g., a particular language filer might easily pick out a collection that are all CC)